bosh-setup/azuredeploy.json

```diff
@@ -457,7 +515,7 @@
           "osDisk": {
             "name": "osdisk",
             "vhd": {
-              "uri": "[concat('http://',variables('defaultStorageAccountName'),'.blob.',variables('environment').serviceHostBase,'/',variables('vmStorageAccountContainerName'),'/',parameters('vmName'),'-osdisk.vhd')]"
+              "uri": "[concat('http://', variables('defaultStorageAccountName'), '.blob.', variables('environment').serviceHostBase, '/', variables('vmStorageAccountContainerName'), '/', parameters('vmName'), '-osdisk.vhd')]"
```

Comment:
Any reason we can't follow the best practices for uri's here: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md  (see #9)